.dotnet/src/Custom/FineTuning/FineTuningJob.cs

```diff
@@ -0,0 +1,23 @@
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJob")]
+public partial class FineTuningJob
+{
+    internal static FineTuningJob FromResponse(PipelineResponse pipelineResponse)
+    {
+        using var document = JsonDocument.Parse(pipelineResponse.Content);
+        return DeserializeFineTuningJob(document.RootElement);
+    }
+
+    public FineTuningJobStatus Status { get; }

```

Comment:
This is just a personal preference thing -- even when I don't change the names, I like to include the explicit `[CodeGenMember("Status")]` attribute; the code generator infers it so long as the name is exactly the same as what it'd generate, but if you renamed later on (e.g. to 'JobStatus', not that it'd make sense here) you'd end up with a painful-to-debug situation where the custom parameter "stopped working" and the old "Status" suspiciously reappeared in generated code. Silent failures are the worst.